tips/views.py

```diff
@@ -1,2 +1,40 @@
-# from django.shortcuts import render
-# Create your views here.
+from django.shortcuts import render, redirect, get_object_or_404
+from .models import Tip
+from .forms import TipForm
+
+
+def board(request):
+    tips = Tip.objects.order_by('-date')
+    return render(request, 'tips/board.html', {'tips': tips})
```

Comment:
You should add a constant URL, like what we did on house, check out our views.py and constants.py, every PR about House should have it.
I think it would make the code a bit nicer :)

Reply:
sure